### DISCUSSION TITLE
All SHAP values negative for a variable (same side of the plot)

### DISCUSSION THREAD

**Discussion by rflameiro** [shap/shap#3353] [Q&A]
Hello, do the results for the first vaiable in this plot seem sensible? Consider that this is a binary variable, and the plot seems to indicate that either the absence or presence of the variable contribute with a negative SHAP value...

![shap_same_side](https://github.com/shap/shap/assets/52053058/86e5d452-cd95-42d5-8b4a-4b35a1874cd5)

**Comment by thatlittleboy** [comment]
This seems to suggest that the samples you are explaining has a different distribution to the samples you trained your model on / background dataset used for the explainer.

Feel free to raise an issue with a reproducible example and more details for us to diagnose the problem
